src/features/rewards/walletWrapper/index.tsx

```diff
@@ -244,6 +245,36 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
     )
   }
 
+  getNotificationButton = (type: NotificationType, onClose: any) => {
+    let buttonText
+    let buttonAction
+
+    switch (type) {
+      case 'grant':
+        buttonText = getLocale('claim')
+        buttonAction = this.onNotificationClick
+        break
+      case 'backupWallet':
+        buttonText = getLocale('backupNow')
+        buttonAction = this.onNotificationClick
+        break
+      default:
+        buttonText = getLocale('OK')
```

Comment:
can we use `ok` and then add `.toUpperCase()` so that we don't need new `ok` locale?

Reply:
done